packages/dev/addons/src/atmosphere/ShadersWGSL/ShadersInclude/atmosphereFragmentDeclaration.fx

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+var<uniform> peakRayleighScattering : vec3<f32>;

```

Comment:
You can remove this file, because it's used when ubos are not supported by the platform, but WebGPU always supports ubos.